.github/workflows/build.yml

```diff
@@ -279,6 +279,36 @@ jobs:
           name: codecov-umbrella
           fail_ci_if_error: false
 
+  build_windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        torch: [1.7.1]
+        include:
+          - torch: 1.7.1
+            torchvision: 0.8.2
```

Comment:
I suggest also including the latest version.

Reply:
done